src/apiProxy/api-proxy.test.js

```diff
@@ -42,18 +42,39 @@ describe('parseRequest', () => {
 });
 
 describe('parseApiResponse', () => {
+	const MOCK_RESPONSE = {
```

Comment:
I have a mild preference to keep this mock in this test file because it's very specific to the response generated by the npm `request` library - not a Hapi `response` or a fetch `response` or an `apiResponse`, which are used elsewhere.